modules/02_welfare/utilitarian/equations.gms

```diff
@@ -78,5 +114,19 @@ q02_inconvPenFeBioSwitch(ttot,regi,entySe,entyFe,te,sector,emiMkt)$((ttot.val ge
 ;
 $ENDIF.INCONV_bioSwitch
 
+*** inconvenience cost for fuel switching between biomass/synfuel in non-energy
+*** use
+q02_inconvPenNonEnSwitch(ttot,regi,entySe,entyFe,te,sector,emiMkt)$(
+                          ttot.val ge cm_startyear
+                      AND se2fe(entySe,entyFe,te) 
+                      AND entyFe2sector2emiMkt_NonEn(entyFe,sector,emiMkt)
+                      AND (entySeBio(entySe) OR entySeSyn(entySe) )        ) ..
+    vm_demFENonEnergySector(ttot,regi,entySe,entyFe,sector,emiMkt)
+  - vm_demFENonEnergySector(ttot-1,regi,entySe,entyFe,sector,emiMkt)
+  + v02_NegInconvPenNonEnSwitch(ttot,regi,entySe,entyFe,sector,emiMkt)
+  - v02_PosInconvPenNonEnSwitch(ttot,regi,entySe,entyFe,sector,emiMkt)
```

Comment:
I suggest putting these two on the right-hand side of the equation.  Makes for better readability.